.github/workflows/arm_workflow.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: arm-workflow
+
+on:
+  workflow_dispatch: null
+  push:
+    branches:
+      - master
+      - 'v1.*'
+  pull_request:
+    branches:
+      - master
+      - 'v1.*'
+    types: [ opened, reopened, synchronize ]
+
+env:
+  MAGMA_VERSION: "1.7.0"
+
+jobs:
+  build-deploy-arm:
+    steps:
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+      - name: Run apt
+        run: sudo apt-get update && sudo apt -y upgrade
+      - name: setup pyenv
```

Comment:
we recently removed the "setup pyenv" step from a couple of workflows because it was not needed - is it needed here?